src/SourceBuild/content/eng/pipelines/vmr-license-scan.yml

```diff
@@ -8,6 +8,8 @@ schedules:
     # Previews don't have internal/release branches so they must be run from non-internal release branches.
     include:
     - main
+    - release/9.0.1xx
+    - release/9.0.1xx-rc*
```

Comment:
```suggestion
    - release/*.0.1xx
    - release/*.0.1xx-rc*
```

Reply:
`release/*.0.1xx` doesn't make sense to me. With the description https://github.com/dotnet/sdk/blob/main/src/SourceBuild/content/eng/pipelines/vmr-license-scan.yml#L7, we should only run `8.0.1xx` from internal/release branches.

Reply:
I just noticed this description myself.  It makes sense to me.  Given that, are any changes even necessary?  This pipeline will run as soon as there is an internal/release/9.0.100-rc1 build.

Reply:
I personally like adding `release/*.0.1xx*`. Can we list `release/8.0.1xx` as an excluded branch?

Reply:
Since there will be an internal/release/9.0.1xx-rc1 build, I will close this PR.